buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/UpdateEapBuildNumber.kt

```diff
@@ -12,12 +12,6 @@ open class UpdateEapBuildNumber : DefaultTask() {
   @Internal
   val intellijSnapshotsUrl: String = "https://www.jetbrains.com/intellij-repository/snapshots/"
 
-  @Internal
-  @Option(
-      option = "fail-if-not-found",
```

Comment:
No longer needed, I'm doing just `git diff-index --quiet HEAD` instead as it's more universal